dotcom-rendering/src/components/Dateline.tsx

```diff
@@ -62,7 +66,7 @@ export const Dateline = ({
 		);
 	}
 	return (
-		<div css={datelineStyles} style={mobileColour}>
+		<div css={datelineStyles} style={mobileColour} data-gu-name="dateline">
```

Comment:
Just looking out of curiosity so feel free to ignore. Do we ever have to care about primary vs. secondary datelines? Whatever that means 😁

Reply:
I was chatting with the Ed Design crew about this earlier and apparently the distinction comes when a piece has been updated. In those cases you have publication date _and_ and last modified date. So I think both pop up.